broker-daemon/models/block-order.js

```diff
@@ -476,6 +438,67 @@ class BlockOrder {
 
     return BlockOrder.fromStorage(blockOrderId, value)
   }
+
+  /**
+   * @param {Array<Order>} orders
+   * @returns {Object} res
+   * @returns {Big} res.inbound
+   * @returns {Big} res.outbound
+   */
+  static async activeAmountsForOrders (orders) {
+    const {
+      CREATED,
+      PLACED,
+      EXECUTING
+    } = OrderStateMachine.STATES
+    const response = {
+      inbound: Big(0),
+      outbound: Big(0)
+    }
+    return orders.reduce((acc, { order, state }) => {
+      // If the order is not in an active state , then we can simply skip it
+      if (![CREATED, PLACED, EXECUTING].includes(state)) {
+        return acc
+      }
+
+      if (state === OrderStateMachine.STATES.EXECUTING) {
```

Comment:
[nit] `EXECUTING` is destructured above, so should be able to use the more concise definition.